tsconfig-build.json

```diff
@@ -24,7 +24,7 @@
         "typeRoots": [
             "node_modules/@types/"
         ],
-        "types": ["node"],
+        "types": ["node", "webappsec-credential-management"],
```

Comment:
do we need a new release because of this change?

Reply:
To fix the issue with Angular 7/8  new release is not required. 
`okta-angular 5.1.1` can work with `okta-auth-js 6.2.0`, user still needs to add `webappsec-credential-management` in `tsconfig` of Angular 7/8 application.

Reply:
looks like this one is the sdk config file, do we also need the type here?

Reply:
We use Angular 7 (as the oldest version we support) as dev dependency: https://github.com/okta/okta-angular/blob/master/package.json#L57
When we update okta-auth-js dev dependency to 6.2.0, sdk build [will fail](https://app.travis-ci.com/github/okta/okta-angular/builds/248190510#L421-L424) without adding type here